civihr_default_theme/scss/pages/_my-leave-request.scss

```diff
@@ -0,0 +1,20 @@
+$row-margin-bottom: 15px;
```

Comment:
Just realized this: this is not a page, it's a component. Hence it should be in the components/ folder. Also the name of the component (ignoring the prefix) is `leave-request-form`, while the name of the file is `my-leave-request`. Please use the name of the component for the name of the file